src/intern/intern-saucelabs.js

```diff
@@ -0,0 +1,30 @@
+define([
+	'./intern'
+], function (intern) {
+
+    intern.capabilities = {
+        project: '<%- appName %>',
+        name: '<%- appName %>',
+        fixSessionCapabilities: false
+    };
+
+    intern.environments = [
+        { browserName: 'internet explorer', version: [ '10.0', '11.0' ], platform: 'Windows 7' },
+        // { browserName: 'microsoftedge', platform: 'Windows 10' },
+        { browserName: 'firefox', version: '43', platform: 'Windows 10' },
+        { browserName: 'chrome', platform: 'Windows 10' },
+        // { browserName: 'safari', version: '9', platform: 'OS X 10.11' },
+        { browserName: 'android', platform: 'Linux', version: '4.4', deviceName: 'Google Nexus 7 HD Emulator' }// ,
```

Comment:
I commented this out actually because I don't believe the issue with android on saucelabs have been resolved yet.